### DISCUSSION TITLE
How to clear results and work dir ?

### DISCUSSION THREAD

**Discussion by Grelot** [nextflow-io/nextflow#2958] [Q&A]
Hi everyone ?

Can nf-tools or nextflow clear all the temporary files generated by it ?

**Comment by manuelesimi** [comment]
For executions where the workdir is kept locally, you can add the following setting to your config file:
```
cleanup = true
```
This will clean up all the task dirs after the pipeline has completed. I don't believe this is supported across all the executions.
